grails-app/i18n/messages_es_MX.properties

```diff
@@ -0,0 +1 @@
+default.number.format=\# \# \#, \# \# \#, \# \# \#0.00
```

Comment:
is this being used yet?

Reply:
This was an "initial" request from the ticket, so I just added it here to not leave it empty